CHANGES.md

```diff
@@ -15,6 +15,8 @@ Format of the entries must be.
 
 ### What's new
 
+* Add root capabilities to dcos-diagnostics (DCOS_OSS_3877)
```

Comment:
/Add root capabilities to dcos-diagnostics/Run dcos-diagnostics as the `root` user./